internal/ls/completions.go

```diff
@@ -1447,7 +1447,7 @@ func (l *LanguageService) getCompletionData(
 		completionKind = CompletionKindMemberLike
 		// Declaring new property/method/accessor
 		isNewIdentifierLocation = true
-		if contextToken.Kind == ast.KindAsteriskToken {
+		if contextToken != nil && contextToken.Kind == ast.KindAsteriskToken {
```

Comment:
We don't need to do all of those defensive checks in Strada, so we should not need them here, just need the actual fix that makes the `contextToken` not `nil`.